---
title: "'Throw' işleneni 'System.Exception' öğesinden türetilmelidir"
ms.date: 07/20/2015
f1_keywords:
- vbc30665
- bc30665
helpviewer_keywords:
- BC30665
ms.assetid: 7c228087-39ea-4b30-a410-6ba711e67e5e
ms.openlocfilehash: abaef1ac3572da1c8270bdd4717876d9b97c91e7
ms.sourcegitcommit: bef803e2025642df39f2f1e046767d89031e0304
ms.translationtype: MT
ms.contentlocale: tr-TR
ms.lasthandoff: 02/15/2019
ms.locfileid: "56303237"
---
# <a name="throw-operand-must-derive-from-systemexception"></a>'Throw' işleneni 'System.Exception' öğesinden türetilmelidir
Bağımsız değişken için sağlanan `Throw` örneğini olması gerektiği `System.Exception` veya türetilmiş bir sınıf örneği `System.Exception`.  
  
 **Hata Kimliği:** BC30665  
  
## <a name="to-correct-this-error"></a>Bu hatayı düzeltmek için  
  
-   Türetilen bir bağımsız değişkenini kullanın `System.Exception`, aşağıdaki örnekte olduğu gibi.  
  
    ```vb
    Throw New System.Exception("This is an error.")  
    ```  
  
## <a name="see-also"></a>Ayrıca bkz.
- [Throw Deyimi](../../visual-basic/language-reference/statements/throw-statement.md)
- [Try...Catch...Finally Deyimi](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
- <xref:System.Exception?displayProperty=nameWithType>
- [İşleme ve .NET özel durumları atma](../../standard/exceptions/index.md)
